packages/lib/src/stories/Paywall.stories.ts

```diff
@@ -1,50 +1,55 @@
-import { html } from "lit-html";
-import type { Meta, StoryObj } from "@storybook/web-components";
-import { Paywall } from "@sesamy/sesamy-components";
-
+import { html } from 'lit-html';
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { Paywall } from '@sesamy/sesamy-components';
 
 interface PaywallProps {
   'settings-url': string;
 }
 
 const meta: Meta<PaywallProps> = {
-  title: "Components/Paywall",
-  tags: ["autodocs"],
-  component: "sesamy-paywall",
-  render: (args) => html` <sesamy-paywall settings-url=${args["settings-url"]}></sesamy-paywall> `,
+  title: 'Components/Paywall',
+  tags: ['autodocs'],
+  component: 'sesamy-paywall',
+  render: (args) =>
+    html` <div style="display: flex; gap: 10px; padding: 10px; --sesamy-main-color: #E71104;">
```

Comment:
I added the red color as a css variable here instead as the red color might be a bit aggressive as a default main color for all vendors.

Reply:
Yes indeed. The red I provided as main was "fokus red" as an example.

The color will/should come from each vendors' individual vendor settings eventually.

The paywall will soon also have colors settings itself, which we also should load in Base.